open_tms_backend/open_tms_close_ticket/index.js

```diff
@@ -1,4 +1,43 @@
 import { shared } from '@appblocks/node-sdk'
+import { Ticket, TicketRevision, TicketActivity } from './models'
```

Comment:
Appblocks intents to build completely mutually exclusive functions that can combine together easily. So importing directly the models is kind of an antipattern. Here all the models are created as shared blocks. You can use our node-sdk to dynamically import shared modules from entire package block. You can refer the docs [here](https://docs.appblocks.com). Meanwhile we have updated, how to use shared blocks, in our contribution.md file as well.